docs/angular.md

```diff
@@ -10,11 +10,13 @@ There is a lot of information out there. Some good. Some bad. But here you'll fi
 ## Table of Contents
 1. [Installing Angular CLI](#installing-angular-cli)
 2. [Starting A New Project](#starting-a-new-project)  
-3. [Routing](#routing)
-4. [Webpack Bundle Analyzer](#webpack-bundle-analyzer)
-5. [Using ngRx with a loading indicator](#using-ngrx-with-a-loading-indicator)
+3. [Starting A New Universal Project](#starting-a-new-universal-project)
+4. [Add TransferState](#add-transferstate)
+5. [Routing](#routing)
+6. [Webpack Bundle Analyzer](#webpack-bundle-analyzer)
+7. [Using ngRx with a loading indicator](#using-ngrx-with-a-loading-indicator)
 
-### Installing Angular CLI
+### 1) Installing Angular CLI
```

Comment:
Numbering is good. But adding in the number is a maintenance issue, every time we add a new thing we will need to update the numbers accordingly.

Reply:
I want to separate the sections because it's kind of difficult to see the hierarchy as you scroll down. I'll take the numbers off and add lines underneath instead.